### PR TITLE
Remove return statement from cli.js

### PR DESCRIPTION
**Pull request by ysangkok**

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -4,8 +4,10 @@ var errno = require('./')
   , arg   = process.argv[2]
   , data, code
 
-if (arg === undefined)
-  return console.log(JSON.stringify(errno.code, null, 2))
+if (arg === undefined) {
+  console.error(JSON.stringify(errno.code, null, 2));
+  process.exit(1);
```
Comment (ralphtheninja) [inline cli.js]: Why `console.error` and why `process.exit(1)`? This is a feature of the cli tool.

Comment (ralphtheninja) [inline cli.js]: Also, semi colons etc.

**Comment by ysangkok** [comment]
@ralphtheninja How about this? :)

**Comment by ralphtheninja** [comment]
0.1.5

**Pull request merged**